test/functional/terminal/highlight_spec.lua

```diff
@@ -165,24 +165,50 @@ end)
 
 describe('synIDattr()', function()
   local screen
-
+  -- Salmon #fa8072
+  -- Maroon #800000
   before_each(function()
     clear()
     screen = Screen.new(50, 7)
-    execute('highlight Normal ctermfg=1 guifg=#ff0000')
+    execute('highlight Normal ctermfg=1 guifg=#ff0000 guibg=Black')
+    execute('highlight Keyword ctermfg=2 guifg=Salmon guisp=Maroon')
   end)
 
   after_each(function()
     screen:detach()
   end)
 
-  it('returns RGB number if GUI', function()
+  it('returns gui-color if GUI', function()
+    eq('1', eval('synIDattr(hlID("Normal"), "fg#")'))
+    eq('-1', eval('synIDattr(hlID("Normal"), "bg#")'))
+
+    eq('2', eval('synIDattr(hlID("Keyword"), "fg#")'))
+    eq('', eval('synIDattr(hlID("Keyword"), "sp#")'))
```

Comment:
shouldn't these 4 lines test `foo`, not `foo#` (which is already tested in the next `describe()`)?